client/src/components/FieldTag/index.tsx

```diff
@@ -20,10 +20,6 @@ const FieldTag: FC<FieldTagProps> = ({
   ...rest
 }): ReactElement => {
   const toClassName = (str: string): string => {
-    if (name !== '') {
```

Comment:
Why was this removed?

Reply:
For the field tags with pronouns I noticed when the name and content didn't match it didn't apply the styling for the tag to have the appropriate color. For example if the name was assigned "she/her" and the content was "pronouns", with this it would check if the name was not an empty string and assign the `className` to ''. I removed it so it wouldn't return '' in `toClassName` and apply the color to the tag using the content name.